pylatex/quantities.py

```diff
@@ -39,7 +39,8 @@ class Quantity(Command):
     packages = [Package('siunitx')]
 
     def __init__(self, quantity, *, options=None, format_cb=None):
-        r"""
+        r"""Initialize a quantity.
```

Comment:
I have all the dev_requirements installed, but it still seems to still complain about the __init__ docstrings.